worker/reference/environment/index.md

```diff
@@ -17,6 +17,7 @@ breadcrumbs:
     <li><a href="#maximum_run_time_per_worker">Maximum Run Time per Worker</a></li>
     <li><a href="#priority_queue_management">Priority Queue Management</a></li>
     <li><a href="#maximum_scheduled_tasks_per_project">Maximum Scheduled Tasks per Project</a></li>
+    <li><a href="#image_size_limitation">Image Size Limitation</a></li>
```

Comment:
I think it's better to mention the word "docker" at least in one place: Docker Image Size Limitation